apps/RiiTag RPC/install-64

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+install-if-not-installed ArmCord
```

Comment:
`install-if-not-installed` is not a function. it is an argument that you can input when calling the `manage` script

refer to the other pi-apps scripts for using it properly and don't forget to add the error if it fails as the other scripts have done https://github.com/search?q=repo%3ABotspot%2Fpi-apps%20install-if-not-installed&type=code

Reply:
@libewa not resolved. please re-read the link above. both the directory and version variables are using illegal formatting

Reply:
i don't get what you're complaining about